content/blog/2023-01-05_XMPP_Summit_26.md

```diff
@@ -0,0 +1,44 @@
+---
+title: XMPP Summit 23
+date: 2023-01-05
+author: XMPP Standards Foundation
+categories: ["XMPP Summit"]
+---
+
+The XMPP Standards Foundation (XSF) will hold its 26th XMPP Summit in Brussels, Belgium this year again! These are the two days preceding [FOSDEM 2024](https://xmpp.org/2024/01/xmpp-at-fosdem-2024/).
+The XSF invites everyone interested to attend, and discuss all things XMPP in person and remote!
+
+### Time & Address
+
+The venue will take place at the Thon Hotel EU including coffee break (from 08:30 o'clock) and lunch (12:00 to 14:00 o'clock) paid by the XSF in the hotel restaurant. 
+
+Date: Thursday, 1st and Friday, 2nd February 1st 2024
+Time: 09:00 - 17:00 o'clock (CET) (both days)
+
+[Thon Hotel EU](https://www.thonhotels.com/de/hotels/belgien/brussel/thon-hotel-eu/) 
+Room: FRANCE  
+Wetstraat / Rue de la Loi 75  
+1040 Brussels
+[Openstreetmap](https://osm.org/go/0EoS9cWe0?layers=N&m=)
+
+Furthermore, the XSF will have its dinner Thursday night which is paid for XSF members. Everyone else is of course invited to participate, however on their own expense. Please reach out if you are participating as none XSF member (see list below).
+
+### Participation
+
+**Please compulsory register before Mo, 15th January 2024!**
+
+Please note that, although we welcome everyone to join, you must announce your attendance beforehand, as the venue is not publicly accessible. If you're interested in attending, 
+please make yourself known by filling out your details on the [wiki page for Summit 26](https://wiki.xmpp.org/web/Conferences/Summit_26). To edit the page, reach out to an XSF member 
+to enter and update your details or you'll need a wiki account, which we'll happily provide for you. Reach out in the [XSF public chatroom](https://xmpp.org/chat#converse/room?jid=xsf@muc.xmpp.org).
+When you sign-up please book the accomodation and your travel. Please also unsign if you will not attend anymore.
+
+### Communication
+
+We kindly ask to be available and check frequently if we have new announcements. So, if you haven't already, make sure that you're signed up to the [Summit mailling list](https://mail.jabber.org/mailman/listinfo/summit) and the [Summit chatroom](xmpp:summit@muc.xmpp.org?join), which is where you can expect most updates to be announced.
+
+Spread the word also via our communication channels such as [Mastoston](https://fosstodon.org/@xmpp) and [Twitter](https://twitter.com/xmpp).
+
+  
+We are really exited seeing so many people already signing up. Looking forward to all of you!  
+  
+The XSF Board
```

Comment:
```suggestion
```
(minor, but I don't know if this is really from "the board" vs. "the XSF" as a whole)

Reply:
@mwild1 see this again: https://github.com/xsf/xmpp.org/pull/1338/commits/cd90f02a844636654c173fbb2543eae739984df6

Reply:
:+1: